src/magic_folder/cli.py

```diff
@@ -118,21 +118,21 @@
     read_config,
 )
 
-from ..frontends.magic_folder import (
+from magic_folder.frontends.magic_folder import (
     MagicFolder,
     load_magic_folders,
     save_magic_folders,
     maybe_upgrade_magic_folders,
 )
-from ..web.magic_folder import (
+from web.magic_folder import (
```

Comment:
I think this definitely should be either `from .web...` or `from magic_folder.web...`.  *Implicit* relative imports are awful.